third_party/rules_pycross/pycross/private/wheel_library.bzl

```diff
@@ -135,3 +174,17 @@ This option is required to support some packages which cannot handle the convers
         ),
     },
 )
+
+def py_wheel_library(patches = None, patch_tool = None, patch_tool_target = None, **kwargs):
+    # If the user has specified patches, but hasn't specified how to apply
+    # those patches, default to our internally-provided patch binary.
+    if patches and not patch_tool and not patch_tool_target:
+        fail("Patches specified, but missing `patch_tool` or `patch_tool_target` attributes. " +
+             "I.e. unsure how to apply patches.")
+
```

Comment:
Thinking out loud, we set it here so that we get the error at load time?

I think just doing it at the top of the `_py_wheel_library` may be just as good. That way we can avoid the macro. On the other hand, if this is idiomatic and something that is commonly done in the bazel world, I am OK with leaving it as is.

Reply:
Yeah, putting the error at load time makes it more likely for the user to notice before uploading for code review. But I don't think it's _really_ buying us anything. The tool itself will fail during the action phase.